MID Server/MID Server Script Includes/IllumioHTTPClient.js

```diff
@@ -41,15 +44,15 @@ IllumioHTTPClient.prototype = {
      **/
     _getProxy: function(proxyFlag) {
         var proxy = null;
-        if (ms.getConfigParameter("mid.proxy.use_proxy") != "true") {
-            this.logger._debug("Proxy is not configured, continuing with internet connection");
+        if (ms.getConfigParameter('mid.proxy.use_proxy') != 'true') {
```

Comment:
This is added because the method is in ServiceNow and it returns string.

Reply:
I understand that the SNOW values are strings, but the inconsistent handling becomes confusing - it would be good to have a helper to convert these boolean strings.

This is something we can target in a refactor for the next major release.